vmck/api.py

```diff
@@ -30,6 +30,8 @@ def create_job(request):
     options.setdefault('memory', 512)
     options.setdefault('image_path', 'imgbuild-master.qcow2.tar.gz')
     options.setdefault('name', 'default')
+    if options['name'] is None:
+        options['name'] = 'default'
```

Comment:
```python
options['name'] = options.get('name') or 'default'
```
It's better because:
* it avoids an `if` (there's a whole school of thought on avoiding IFs in the code entirely)
* it covers `""`
* it covers the case when the value is missing entirely from `options`, because hey, maybe we want to preserve backwards compatibility :)